### PR TITLE
Made Discord Bot message able to be turned off.

### PR DESCRIPTION
**Pull request by KozmikNano**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ try {
 		process.exit(1);
 	}
 }
+var Bot;
```
Comment (rob9315) [inline main.js:25]: Do not use a local variable for a value that is always accessible through the config object.
(Variables should also not start with capitaziled letters, those indicate classes and types.)

File: main.js [hunk 2]
```diff
@@ -52,7 +53,7 @@ const askForSecrets = async () => {
 		if(err.code != "ENOENT") throw err;
 	}
 	let canSave = false;
-	if(!(config.has("username") && config.has("mcPassword") && config.has("updatemessage") || config.has("profilesFolder"))) {
+	if(!(config.has("username") && config.has("mcPassword") && config.has("Bot") && config.has("updatemessage") || config.has("profilesFolder"))) {
```
Comment (rob9315) [inline main.js:56]: Bot is not very descriptive

**Comment by rob9315** [comment]
Bot 0, 1 and 2 aren't very descriptive options. There is already a property for if to use the discord bot (discordBot), so the option should rather be discordMessages true|false. discordBot should automatically be set to false if the Token is a blank string.

**Comment by KozmikNano** [comment]
So you just want me to rename the Bot variable?

File: main.js [hunk 3]
```diff
@@ -93,7 +104,9 @@ const askForSecrets = async () => {
 	if (config.get("discordBot")) {
 		dc = new discord.Client();
 		dc.login(discordBotToken).catch(()=>{
-			console.warn("There was an error when trying to log in using the provided Discord bot token."); //handle empty tokens gracefully
+			if (config.Bot != "1"){
```
Comment (rob9315) [inline main.js:107]: Change this check to 
```suggestion
			if (config.discordBot){
```

**Review by rob9315 — CHANGES_REQUESTED** [review]

**Comment by rob9315** [comment]
Sorry, no, I took a while to review

File: main.js [hunk 4]
```diff
@@ -73,10 +74,20 @@ const askForSecrets = async () => {
...
+			discordBotToken = await promisedQuestion("BotToken: ");
+			localConf.BotToken = discordBotToken;
+			Bot = "2";
+			localConf.Bot = Bot;
+		}	else {
+			Bot = "1";
+			localConf.Bot = Bot;
+			localConf.BotToken = "0";
+		}
+
```
Comment (rob9315) [inline main.js:90]: Please change this logic to make better use of the existing config options like I described in a previous comment

**Review by rob9315 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**